src/SubmitPage/SubmitForm.js

```diff
@@ -30,7 +30,7 @@ const {
   SAFETY_CHECKS,
 } = FORM_FIELDS;
 
-const enableSafetyChecks = false;
+const enableSafetyChecks = true;
```

Comment:
is this intentional ?

Reply:
yeah, enabling the feature. backend changes are in prod